charon/src/ast/expressions.rs

```diff
@@ -484,6 +484,8 @@ pub enum RawConstantExpr {
     /// representation isn't possible (e.g. for unions) or just isn't implemented yet.
     #[drive(skip)]
     RawMemory(Vec<u8>),
+    /// A constant expression that Charon still doesn't handle, along with the reason why.
+    Opaque(String),
```

Comment:
I prefer to skip here rather than in the visitor trait definition, because for cases like the `Vec<u8>` above, the visitor-wide `skip` would still iterate through the whole `Vec`.
```suggestion
    #[drive(skip)]
    /// A constant expression that Charon still doesn't handle, along with the reason why.
    Opaque(String),
```